exotations/solvers/ompl_solver/init/OMPLsolver.in

```diff
@@ -2,4 +2,12 @@ extend <exotica/MotionSolver>
 Required std::string Solver;
 Optional std::string Range = "1";
 Optional bool Smooth = true;
-Optional double Margin = 0.01;
+Optional double Margin = 2.0;
```

Comment:
I believe this undoes the previous PR.
Otherwise good to go.

Reply:
Yes, on purpose - the damping will take care of it slightly and otherwise the operator should specify it in the XML.